data_management.py

```diff
@@ -230,3 +230,25 @@ def set_monthly_income(data: dict):
     print()
 
     return data
+
+def set_category_budget(data: dict):
+    categories = ["Food", "Rent", "Utilities", "Transport"]
```

Comment:
what if user have another category , not in boundary. So, I suggest to use groupby function

Reply:
Changed the definition to `df[df["Type"] == "Expense"]["Category"].unique().tolist()`.
Now the category of the record in the DataFrame whose "Type" is "Expense" is retrieved.